feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.spec.ts

```diff
@@ -155,6 +162,12 @@ describe('ConfigProductTitleComponent', () => {
             useClass: MockProductService,
           },
           { provide: IconLoaderService, useClass: MockIconFontLoaderService },
+          {
+            provide: FeaturesConfig,
+            useValue: {
+              features: { level: '3.1' },
```

Comment:
can we already 3.3 here? (most likely we release with 3.3)

Reply:
Done!